docs/site/Fields-filter.md

```diff
@@ -0,0 +1,86 @@
+---
+title: 'Fields filter'
+lang: en
+keywords: LoopBack 4.0, LoopBack 4, fields
+layout: readme
+source: loopback-next
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Fields-filter.html
+---
+
+A _fields_ filter specifies properties (fields) to include or exclude from the results.
+
+### REST API
+
+
+**The <true|false> does not work for url as the boolean is being concerted to string, which is always trru**
+<pre>
+filter[fields][<i>propertyName</i>]=<true>&filter[fields][<i>propertyName</i>]=<true>...
+</pre>
+
+Note that to include more than one field in REST, use multiple filters.
+
+You can also use [stringified JSON format](Querying-data.html#using-stringified-json-in-rest-queries) in a REST query.
+
+### Node API
+
+{% include content/angular-methods-caveat.html lang=page.lang %}
+
+<pre>
+{ fields: {<i>propertyName</i>: <true|false>, <i>propertyName</i>: <true|false>, ... } }
+</pre>
+
+Where:
+
+* _propertyName_ is the name of the property (field) to include or exclude.
+* `<true|false>` signifies either `true` or `false` Boolean literal. Use `true` to include the property or `false` to exclude it from results.
+
+By default, queries return all model properties in results. However, if you specify at least one fields filter with a value of `true`,
+then by default the query will include **only** those you specifically include with filters.
+
+### Examples
+
+Return customer information only with `name` and `address`, and hide their `id`:
+
+**REST**
+
+**<false> does not wort in url for now as it is converted to string**
+should be:
+`/customers?filter[fields][id]=false&filter[fields][name]=true&filter[fields][address]=true`
+But now is:
+`/customers?filter[fields][name]=true&filter[fields][address]=true`
+
+{% include code-caption.html content="Node API" %}
```

Comment:
Content should be a file name

Reply:
I think `code-caption.html` is to have a different layout for the content. 
![image](https://user-images.githubusercontent.com/50331796/78811347-bb712b00-7997-11ea-9651-ce8044af5c54.png)

Personally I like the 'Node API' one more, WDYT?

Reply:
Because if you check the LB3 file: https://loopback.io/doc/en/lb3/Where-filter.html, it mixes the bold font with the bold title, which is not that readable. (see the `equivalence` and `RESR` in https://loopback.io/doc/en/lb3/Where-filter.html#examples)

Reply:
If that's okay with you, I am thinking to use the layout for the title of each example. i.e:
```
{% include code-caption.html content="REST" %}
rest example...
{% include code-caption.html content="Node API" %}
node example...
```

Reply:
good with me 👍